src/js/components/List/List.js

```diff
@@ -5,21 +5,17 @@ import { Box } from '../Box';
 import { InfiniteScroll } from '../InfiniteScroll';
 import { Keyboard } from '../Keyboard';
 import { Text } from '../Text';
-import { focusStyle, genericStyles } from '../../utils';
+import { focusStyle, genericStyles, useForwardedRef } from '../../utils';
 
 const StyledList = styled.ul`
   list-style: none;
   ${props => !props.margin && 'margin: 0;'}
   padding: 0;
   ${genericStyles}
 
-  ${props =>
-    props.onClickItem &&
-    `
-    &:focus {
-      ${focusStyle({ forceOutline: true, skipSvgChildren: true })}
-    }
-  `}
+  &:focus {
+    ${props => props.tabIndex >= 0 && focusStyle({ skipSvgChildren: true })}
```

Comment:
Are you sure we don't want `forceOutline: true`?

Reply:
What is the difference of using forceOutline vs not? What I noticed was that it wasn't applying the same box-shadow styling as the typical focus indicator. instead, it was a solid border with no fuzziness around the edges.

Reply:
I'm looking at `focusStyles` and seeing that by adding `forceOutline` we are entering this block of code, but I am not fully understanding the rationale for using it vs not using it: https://github.com/grommet/grommet/blob/master/src/js/utils/styles.js#L217

Reply:
If the theme wants the focus to be drawn via a border, this flag indicates that for this situation, we should use an `outline` instead, which is what I think we want here, since the `<ul />` doesn't have a border itself.

Reply:
Thanks for the clarification. I see this better in the code now.